pcmanfm/desktopwindow.cpp

```diff
@@ -1523,6 +1584,17 @@ bool DesktopWindow::eventFilter(QObject* watched, QEvent* event) {
             break;
         case QEvent::Wheel:
             // removal of scrollbars is not enough to prevent scrolling
+            if(passWheelToRoot_) {
+                QWheelEvent* e = static_cast<QWheelEvent*>(event);
+                // Forward wheel events we received to the root window.
+                // check if the user is over a blank area
+                QModelIndex index = listView_->indexAt(e->pos());
+                if(!index.isValid()) {
+                    forwardWheelEventToRoot(e);
+                    return true;
+                }
+            }
+            break;
             return true;
```

Comment:
Please correct this: `return true;` isn't reached when `passWheelToRoot_` is `false`. Long story short, `return true;` is needed with item DND.

Reply:
Done.

Reply:
Thanks!